<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrogViewer</class>
 <widget class="QWidget" name="FrogViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="rec_button">
     <property name="text">
      <string>Rec</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="next_button">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="gif_button">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="previous_button">
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="glow_wire_button">
     <property name="text">
      <string>Glow Wire</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="close_button">
     <property name="text">
      <string>Close Frog</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
